frontend/src/metabase/common/components/FilterPicker/SimpleLayout.tsx

```diff
@@ -0,0 +1,53 @@
+import type { ReactNode } from "react";
+import { isValidElement } from "react";
+import { t } from "ttag";
+import { Button, Box } from "metabase/ui";
+import { MAX_WIDTH } from "./constants";
+import { BackButton } from "./BackButton";
+import { Header } from "./Header";
+import { Footer } from "./Footer";
+
+interface SimpleLayoutProps {
+  columnName: string;
+
+  isNew: boolean;
+  canSubmit: boolean;
+
+  headerRight?: ReactNode;
+  footerLeft?: ReactNode;
+  children: ReactNode;
+  testID?: string;
+
+  onSubmit: () => void;
+  onBack: () => void;
+}
+
+export function SimpleLayout({
+  columnName,
+  isNew,
+  canSubmit,
+  headerRight,
+  footerLeft,
+  children,
+  testID,
+  onSubmit,
+  onBack,
+}: SimpleLayoutProps) {
+  return (
+    <Box maw={MAX_WIDTH} data-testid={testID}>
+      <Header>
```

Comment:
This generic component makes me worry about potential changes in the future. WDYT about extending current `Header` and `Footer` components for the same purpose? And you could pass `children`

Reply:
Yep, but you could name slots as `children` I think

Reply:
@kulyk ^

Reply:
@ranquild how about that [`824c66f`](https://github.com/metabase/metabase/pull/35385/commits/824c66ffcdb7455dacdff320497c78bc94e41275)?

Reply:
@kulyk LGTM, but I'd remove `SimpleLayout` altogether and use new `Header` and `Footer` directly. I anticipate issues with extending if we leave `SimpleLayout`.

Reply:
@ranquild, please take another look [`d57f2ac`](https://github.com/metabase/metabase/pull/35385/commits/d57f2ac50e36044652167c7f210e37851ff77abb)